crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -214,6 +217,10 @@ impl Plugin for VisibilityPlugin {
                 .after(UpdateProjectionFrusta)
                 .after(VisibilityPropagate)
                 .after(TransformSystem::TransformPropagate),
+        )
+        .add_system_to_stage(
+            CoreStage::Last,
```

Comment:
(I'm new to reviewing)

is there a reason this has to be CoreStage::Last vs PostUpdate (but after CheckVisibility?)

Reply:
I didn't put much thought into this. I was worried about entity spawning in Stage X and components being added in Stage Y. If we read the ECS for inconsistency between the two stages, we log a false positive. So I chose `CoreStage::Last`. But it's still possible for X to be AFTER Y, then it will be inconsistent during `Last`.